docs/source/server-process.md

```diff
@@ -126,6 +126,12 @@ the following keys:
    JupyterLab launcher
 3. **title**
    Title to be used for the launcher entry. Defaults to the name of the server if missing.
+4. **path_info**
+   The trailing path that is appended to the user's server URL to access the proxied server.
+   By default it is the name of the server followed by a trailing slash.
+5. **category**
+   The category for the launcher item. Currently only used by the JupyterLab launcher.
+   By default it is "Notebook".
```

Comment:
I think if we can indicate here that icons won't work unless category is 'Notebook' or 'Kernel', we can get this merged as is!